Warning from ANTsR

### Operating system and version

Ubuntu   24.04.4   LTS

### CPU architecture

x86_64 (PC, Intel Mac, other Intel/AMD)

### ANTs code version

ANTsRCore

### ANTs installation type

Other (please specify below)

### Summary of the problem

https://github.com/ANTsX/ANTsRCore/actions/runs/27302796832/job/80652690060#step:6:79

### Commands to reproduce the problem.

https://github.com/ANTsX/ANTsRCore/actions/runs/27302796832/job/80652690060#step:6:79

### Output of the command with verbose output.

  /home/runner/work/ANTsRCore/ANTsRCore/check/ANTsRCore.Rcheck/00_pkg_src/ANTsRCore/src/ants/Examples/antsApplyTransforms.cxx:298:28: warning: array subscript 3 is outside array bounds of ‘itk::ImageBase<3>::SizeType [1]’ {aka ‘itk::Size<3> [1]’} [-Warray-bounds=]


### Data to reproduce the problem

https://github.com/ANTsX/ANTsRCore/actions/runs/27302796832/job/80652690060#step:6:79

--- a/Examples/antsApplyTransforms.cxx
+++ b/Examples/antsApplyTransforms.cxx
@@ -301,9 +301,9 @@ antsApplyTransforms(itk::ants::CommandLineParser::Pointer & parser, unsigned int
       typename TimeSeriesImageType::SizeType size = extractRegion.GetSize();
 
       // Check if the extractTimeIndex is within range
-      if (extractTimeIndex >= size[3])
+      if (extractTimeIndex >= size[Dimension])
       {
-        std::cerr << "Error: time index to extract is out of range [0, " << (size[3] - 1) << "]" << std::endl;
+        std::cerr << "Error: time index to extract is out of range [0, " << (size[Dimension] - 1) << "]" << std::endl;
         return EXIT_FAILURE;
       }
 

